.env.miniserver.example

```
@@ -0,0 +1,7 @@
+VITE_BACKEND_BASE_URL=https://fairplay.rktclgh.site
+VITE_FRONTEND_BASE_URL=https://fairplay.rktclgh.site
+VITE_CDN_BASE_URL=https://fairplay.rktclgh.site/uploads
+VITE_ENABLE_NEW_PICKS=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_ENABLE_NEW_PICKS key should go before the VITE_FRONTEND_BASE_URL key

(UnorderedKey)